apps/web/src/stores/experiments.ts

```diff
@@ -107,34 +107,40 @@ export function useExperiments(
   const { execute: create, isPending: isCreating } = useLatitudeAction(
     createExperimentAction,
     {
-      onSuccess: async ({ data: experiment }: { data: Experiment }) => {
-        mutateCount((prev) => (prev ? prev + 1 : 1), {
+      onSuccess: async ({ data: experiments }: { data: Experiment[] }) => {
+        mutateCount((prev) => (prev ?? 0) + experiments.length, {
           revalidate: false,
         })
 
-        const experimentDto = {
-          ...experiment,
-          results: {
-            passed: 0,
-            failed: 0,
-            errors: 0,
-            totalScore: 0,
-          },
-        } as ExperimentDto
+        const experimentDtos = experiments.map(
+          (experiment) =>
+            ({
+              ...experiment,
+              results: {
+                passed: 0,
+                failed: 0,
+                errors: 0,
+                totalScore: 0,
+              },
+            }) as ExperimentDto,
+        )
```

Comment:
hmm this looks like a job for a backend presenter rather than the frontend store